bugsnag/src/main/java/com/bugsnag/Configuration.java

```diff
@@ -46,13 +47,13 @@ public class Configuration {
     public boolean sendThreads = false;
 
     Collection<Callback> callbacks = new ConcurrentLinkedQueue<Callback>();
-    Serializer serializer = new Serializer();
+    Serializer serializer;
     private final AtomicBoolean autoCaptureSessions = new AtomicBoolean(true);
     private final AtomicBoolean sendUncaughtExceptions = new AtomicBoolean(true);
 
     Configuration(String apiKey) {
         this.apiKey = apiKey;
-
+        this.serializer = new DefaultSerializer();
```

Comment:
I can't see any reason why this can't be a public property (defined a little higher-up) and assigned to on the same line as being declared (so it fits the existing style).

Also, we don't need a setter (at least I don't think we do)